hyperapp.d.ts

```diff
@@ -0,0 +1,45 @@
+export as namespace hyperapp
+
+// VNode part
+
+export interface VNode<Data> {
+  tag: string
+  data?: Data
+  children: VElement<{}>[]
+}
+
+export type VElement<Data> = VNode<Data> | string
```

Comment:
Not if you are thinking like the last sentence.

You cant tell a `VirtualNode` could be a `VirtualNode` or a `string`, isnt it ?
But its easy to reason about a `VirtualNode` **and** a `VirtualElement` that could be either a `VirtualNode` or a `string`.

The `VirtualNode` is the vdom thing. And the `VirtualElement` is in most case a children. And a children could really be another `VirtualNode` or a `string`.